templates/adv_im_calc.sl.template

```diff
@@ -51,7 +51,7 @@ if [[ $success == 0 ]]; then
     # tar the temporary outputs from analysis
     # following IM_calc_dir/station/model/component_dir
     # using \; instead of + to avoid extra long exec bash command. (this will result in a exec for every file found
-    find {{sim_IM_calc_dir}}/*/*/ -mindepth 1 -maxdepth 1 -type d -exec tar --remove-files -uvf {{sim_IM_calc_dir}}/adv_im_out.tar -C {{sim_IM_calc_dir}} "{}" \;
+    find {{sim_IM_calc_dir}}/*/*/ -mindepth 1 -maxdepth 1 -type d -exec bash -c "f_path={}; tar --remove-files -uvf {{sim_IM_calc_dir}}/adv_im_out.tar -C {{sim_IM_calc_dir}} \${f_path#{{sim_IM_calc_dir}}/}" \;
```

Comment:
Is that #meant to be there?

That last one might be better as a separate variable - I can't work out what that guy is doing.

Reply:
Yeah, I think having that as a separate variable will help with clarity too, will help document what the variable is for

Reply:
the # is for sub-string removal.

i'll add in a new variable for clarity.